visualization/generate_experimentId.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import numpy as np
+import numbers
+
+
+def generate_experimentId(measurement_data):
```

Comment:
Please at a real test-case for this. I.e. a function which checks that for a well-defined input whether a well-defined output is generated.